messages/faq_waiting.md

```diff
@@ -1,10 +1,10 @@
 ## Waiting time in VATSIM Scandinavia
-The waiting time varies by country. Generally, it may extend up to 12 months or follow the timing specified in your training confirmation email.
+The waiting time for ATC Training varies by country. Generally, it may extend up to 12 months or follow the timing specified in your training confirmation email.
 
 It's impossible to give you a queue number or exact estimate, because the training queue is dynamic and varies with a couple of factors such as:
 - Mentor availability,
 - Which rating a mentor can train,
-- How long existing students use on their training,
-- Other training types that have higher priority
+- How long existing students take to complete their training,,
```

Comment:
```suggestion
- How long existing students take to complete their training,
```
for the record, it dampens my expectations if a docs spelling update includes erroneous commas 😅 

Note to self from the original line: why existing? Why not just _students_? Implied that the time students are training, they must be existing students.

```suggestion
- How long students take to complete their training,
```